gobblin-docs/sinks/ParquetHdfsDataWriter.md

```diff
@@ -0,0 +1,23 @@
+# Description
+
+Writes Parquet records to Parquet data files on Hadoop file systems.
+
+
+# Usage
+
+    writer.builder.class=org.apache.gobblin.writer.ParquetHdfsDataWriter
+    writer.destination.type=HDFS
+
+For more info, see [`ParquetHdfsDataWriter`](https://github.com/apache/incubator-gobblin/blob/master/gobblin-modules/gobblin-parquet/src/main/java/org/apache/gobblin/writer/ParquetHdfsDataWriter.java)
```

Comment:
Minor: I know the class has a good document. I would re-organize the class comment here and provide a better format to read.